run_test.sh

```diff
@@ -42,13 +42,13 @@ echo -e "\033[92m\033[1mDatabase ready. Running tests...\033[0m"
 gcc --version >/dev/null 2>&1
 if [ $? -ne 0 ]; then
 	echo -e "\033[33mgcc is missing. Running tests without data race checking.\033[0m"
-	gotest -v -p 1 -coverprofile=c.out -coverpkg=./... ./... ; go tool cover -html=c.out ; go tool cover -func=c.out | grep total ; rm c.out
+	go test -v -p 1 -coverprofile=c.out -coverpkg=./... ./... ; go tool cover -html=c.out ; go tool cover -func=c.out | grep total ; rm c.out
 else
-	gotest -v -p 1 -race -coverprofile=c.out -coverpkg=./... ./... ; go tool cover -html=c.out ; go tool cover -func=c.out | grep total ; rm c.out
+	go test -v -p 1 -race -coverprofile=c.out -coverpkg=./... ./... ; go tool cover -html=c.out ; go tool cover -func=c.out | grep total ; rm c.out
```

Comment:
https://github.com/rakyll/gotest -> the curl gives 404. I assumed it's no longer maintained. But with go install maybe it will work ;p I will remove it from this PR.